services/movie.py

```diff
@@ -0,0 +1,40 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
```

Comment:
The return type should be `QuerySet[Movie]` instead of `QuerySet` to specify the type of objects in the queryset.